projects/pastanaga/src/lib/dialog/dialog.component.ts

```diff
@@ -91,6 +91,10 @@ export class DialogComponent extends BaseDialogComponent implements OnInit, Afte
         this.closed.next();
     }
 
+    refresh() {
+        markForCheck(this.cdr);
+    }
```

Comment:
would be good to document in dialog README what this method is for and in when circonstances it is useful

Reply:
Sure, I will add it.